Update account_invoice.py

cbssolutions opened the pull request:
Acum modulul transforma si cheltuiala in nedeductibila; pana acum facea doar tva-ul nedeductibil.

cbssolutions committed:
Update account_invoice.py

Acum modulul transforma si cheltuiala in nedeductibila; pana acum facea doar tva-ul nedeductibil.

yoyo2k commented:
3 observaţii:
1. Indentare. Folosește 4 spaţii în loc de tab şi aranjează linia 74 că o să dea eroare
2. Lasă un spaţiu la comentarii între # şi comentariu
3. Lasă spaţiu între =. Ex: a = b, nu a=b 

Aceste îndrumări se găsesc în PEP8 - Style Guide for Python Code

cbssolutions committed:
Update account_invoice.py

1,2,3 =ok

desi cred ca  ar trebui sa punem si taxa in acelasi cont cu cel al produsului pt ca poate fi cheltuiala nedeductibila cu benzia 6022, cu materii prime nedeductibile 6021, dar o sa mai vorbesc cu contabila

feketemihai commented:
:+1:  Ok de la mine... @yoyo2k @filsys @dhongu

feketemihai commented:
:-1: Pardon...nu e ok...schimba contul de la prima linie in factura...daca pui neexigibil pe linia a 2 sau a 3...schimba contul la prima linie..nu la cea care e neexigibila...

feketemihai commented:
Mai ai nevoie si de asta:

```
@api.model
def move_line_get_item(self, line):
    if line.not_deductible:
        return {
            'type': 'src',
            'name': line.name.split('\n')[0][:64],
            'price_unit': line.price_unit,
            'quantity': line.quantity,
            'price': line.price_subtotal,
            'account_id': line.company_id.property_undeductible_account_id.id,
            'product_id': line.product_id.id,
            'uos_id': line.uos_id.id,
            'account_analytic_id': line.account_analytic_id.id,
            'taxes': line.invoice_line_tax_id,
        }
    else:
        return super(account_invoice_line, self).move_line_get_item(line)
```

ca sa suprascrie contul in linia corecta...

si scos partea ce ai facut-o in cealalta functie...

feketemihai commented:
Ai aici codul:

https://github.com/feketemihai/l10n-romania/tree/patch-5

feketemihai commented:
Update in #67
